assets/css/index.css

```diff
@@ -144,12 +132,15 @@
 	background-color:rgba(255, 255, 255, 0.7);
 	overflow-x: wrap;
 	overflow-y: hidden;
-	-webkit-user-select: none; /* Safari */        
-	-moz-user-select: none; /* Firefox */
-	-ms-user-select: none; /* IE10+/Edge */
-    user-select: none; /* Standard */
     background-color: #666;
 }
 .item:hover .price {
 	visibility: visible;
+}
+
+#pricelist {
```

Comment:
Try not to target on an ID directly.

Reply:
what is the problem with targeting ID ?

Reply:
https://stackoverflow.com/questions/8084555/why-selecting-by-id-is-not-recommended-in-css You're basically cornering yourself in, and won't be able to override it later with another selector.